augly/image/transforms.py

```diff
@@ -211,7 +211,35 @@ def apply_transform(
         @returns: Augmented PIL Image
         """
         return F.brightness(image, factor=self.factor, metadata=metadata)
-
+   
+   
+class Skew(BaseTransform):
+    def __init__(self, level: float = 0.5, axis: str = "x-axis", p: float = 1.0):
+        """
+        @param level: the level of skew to apply to the image; a larger value means the skew
+            will be more intense and range between [-2, 2]
```

Comment:
please make the change to this param description suggested in `functional.py` here too

Reply:
Done !!